example/samples/install_debian_payload_full.json

```diff
@@ -0,0 +1,63 @@
+{
+    "options": {
+        "defaults": {
+            "rootPassword": "RackHDRocks!",
+            "hostname": "rackhd",
+            "domain": "rackhd.io",
+	    "osName": "debian",
+	    "version": "stretch",
+	    "baseUrl": "dists/xenial/main/installer-amd64/current/images/netboot/ubuntu-installer/amd64",
+	    "repo": "http://172.31.128.1:8080/debian",
+            "users": [
+                {
+                    "name": "arash",
+                    "password": "123456",
+                    "uid": 1010,
+                    "sshKey": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDJQ631/sw3D40h/6JfA+PFVy5Ofz6eu7caxbv0zdw4fTJsrFcOliHZTEtAvqx7Oa8gqSC6d7v61M0croQxtt1DGUcH2G4yhfdZOlK4pr5McwtX0T/APACdAr1HtP7Bt7u43mgHpUG4bHGc+NoY7cWCISkxl4apkYWbvcoJy/5bQn0uRgLuHUNXxK/XuLT5vG76xxY+1xRa5+OIoJ6l78nglNGrj2V+jH3+9yZxI43S9I3NOCl4BvX5Cp3CFMHyt80gk2yM1BJpQZZ4GHewkI/XOIFPU3rR5+toEYXHz7kzykZsqt1PtbaTwG3TX9GJI4C7aWyH9H+9Bt76vH/pLBIn rackhd@rackhd-demo"
+                }
+            ],
+            "rootSshKey": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDJQ631/sw3D40h/6JfA+PFVy5Ofz6eu7caxbv0zdw4fTJsrFcOliHZTEtAvqx7Oa8gqSC6d7v61M0croQxtt1DGUcH2G4yhfdZOlK4pr5McwtX0T/APACdAr1HtP7Bt7u43mgHpUG4bHGc+NoY7cWCISkxl4apkYWbvcoJy/5bQn0uRgLuHUNXxK/XuLT5vG76xxY+1xRa5+OIoJ6l78nglNGrj2V+jH3+9yZxI43S9I3NOCl4BvX5Cp3CFMHyt80gk2yM1BJpQZZ4GHewkI/XOIFPU3rR5+toEYXHz7kzykZsqt1PtbaTwG3TX9GJI4C7aWyH9H+9Bt76vH/pLBIn rackhd@rackhd-demo",
+            "dnsServers": [
+                "8.8.8.8",
+                "4.2.2.4"
+            ],
+            "networkDevices": [
+                {
+                    "device": "eno1",
+                    "ipv4": {
+                        "ipAddr": "192.168.1.29",
+                        "gateway": "192.168.1.1",
+                        "netmask": "255.255.255.0"
```

Comment:
Please add `ipv6` and `vlanIds`. You could look into the ubuntu full payload for an example.